_build_dependencies.sh

```diff
@@ -2,7 +2,7 @@
 # For dreamchecker
 export SPACEMANDMM_TAG=suite-1.7.1
 # For TGUI
-export NODE_VERSION=12
+export NODE_VERSION=16
```

Comment:
Node 16 stopped receiving security updates 4 days ago. Can we bump this to the next LTS release, 18?

Reply:
sure